native/cocos/bindings/manual/jsb_websocket_server.cpp

```diff
@@ -34,36 +34,33 @@
 #include "cocos/bindings/manual/jsb_global.h"
 #include "cocos/network/WebSocketServer.h"
 
-using namespace cc;
-using namespace cc::network;
+se::Class *__jsb_WebSocketServer_class = nullptr; // NOLINT
+se::Class *__jsb_WebSocketServer_Connection_class = nullptr; // NOLINT
 
-se::Class *__jsb_WebSocketServer_class = nullptr;
-se::Class *__jsb_WebSocketServer_Connection_class = nullptr;
+using WSSPTR = std::shared_ptr<cc::network::WebSocketServer> *;
+using WSCONNPTR = std::shared_ptr<cc::network::WebSocketServerConnection> *;
 
-typedef std::shared_ptr<WebSocketServer> *WSSPTR;
-typedef std::shared_ptr<WebSocketServerConnection> *WSCONNPTR;
+static int sendIndex = 1;
 
-static int __sendIndex = 1;
-
-static ccstd::string gen_send_index() {
+static ccstd::string genSendIndex() {
     char buf[128] = {0};
-    snprintf(buf, 127, "__send_[%d]", __sendIndex++);
+    snprintf(buf, 127, "__send_[%d]", sendIndex++);
     return buf;
 }
 
-static bool WebSocketServer_finalize(se::State &s) {
-    WSSPTR cobj = (WSSPTR)s.nativeThisObject();
+static bool WebSocketServer_finalize(se::State &s) { // NOLINT(readability-identifier-naming)
+    auto *cobj = static_cast<WSSPTR>(s.nativeThisObject());
     CC_LOG_INFO("jsbindings: finalizing JS object %p (WebSocketServer)", cobj);
     return true;
 }
 SE_BIND_FINALIZE_FUNC(WebSocketServer_finalize)
 
-static bool WebSocketServer_constructor(se::State &s) {
+static bool WebSocketServer_constructor(se::State &s) { // NOLINT(readability-identifier-naming)
     const auto &args = s.args();
-    int argc = (int)args.size();
+    int argc = static_cast<int>(args.size());
     if (argc == 0) {
         se::Object *obj = s.thisObject();
-        WSSPTR cobj = ccnew std::shared_ptr<WebSocketServer>(ccnew WebSocketServer());
+        auto *cobj = ccnew std::shared_ptr<cc::network::WebSocketServer>(ccnew cc::network::WebSocketServer());
```

Comment:
Private data is nolonger needed to be wrapped with `shared_ptr` manually. PrivateObject will do this automatically.
```c++
auto *cobj = ccnew cc::network::WebSocketServer();
obj->setPrivateData(cobj);
```

Accessing with 
```c++
    auto *cobj = static_cast<WebSocket*>(s.nativeThisObject());
```

Reply:
done. please have a check.